src/api/common/hybrid/index.js

```diff
@@ -1,36 +1,43 @@
 import testRunTracker from './test-run-tracker';
 import compiledCode from './compiled-code-symbol';
+import replicator from './replicator';
 import TestRun from '../../../test-run';
-import compileHybridFunction from '../../../compiler/es-next/compile-hybrid-function';
-import { ExecuteClientCodeCommand } from '../../../test-run/commands';
+import { compileHybridFunction } from '../../../compiler/es-next/hybrid-function';
+import { ExecuteHybridFunctionCommand } from '../../../test-run/commands';
```

Comment:
I thought we will have `client-code` machinery and hybrid functions on top of them and will use `client-code` machinery in raw API (since we don't have hybrids there). But then I realized that once we will need client code parameters in raw API (and we will need them) we will use hybrids machinery anyway (but it will be implemented a little bit differently for raw API). Therefore, it makes sense to use `hybrid` term in code. Meanwhile, end-user doesn't know about our intrinsics, so we use term "client code" in error message to make them generic (e.g. because we have `Hybrid` and `t.eval`)